apps/render_cf.py

```diff
@@ -213,13 +213,11 @@ def render_default_params_by_job_type(job_types: dict) -> None:
 
 
 def render_costs(job_types: dict, cost_profile: str) -> None:
-    costs = [
-        {
-            'job_type': job_type,
-            **job_spec['cost_profiles'][cost_profile],
-        }
-        for job_type, job_spec in job_types.items()
-    ]
+    costs = {}
+
+    for job_type, job_spec in job_types.items():
+        costs[job_type] = job_spec['cost_profiles'][cost_profile]
```

Comment:
Can simplify even further:
```suggestion
    costs = {job_type: job_spec['cost_profiles'][cost_profile] for job_type, job_spec in job_types.items()}
```